src/answers-umd.js

```diff
@@ -504,15 +504,15 @@ class Answers {
   }
 
   /**
-   * Performs a translation which supports
+   * Processes a translation which includes performing
    * interpolation, pluralization, or both
    * @param {string} translations The translations, or a stringified JSON of possible translations
    * @param {Object} interpolationParams Params to use during interpolation
    * @param {number} count The count associated with the pluralization
    * @returns {string} The translation
```

Comment:
How about "The translation with any interpolation or pluralization applied"?

Reply:
That works